tools/topology/topology2/cavs/platform/intel/deep-buffer.conf

```diff
@@ -0,0 +1,39 @@
+
+	Object.Pipeline {
+		deepbuffer-playback."1" {
+			index $DEEP_BUFFER_PIPELINE_ID
+
+			Object.Widget.copier.1 {
+				stream_name 'Deepbuffer Playback'
+			}
+
+			Object.Widget.gain.1 {
+				Object.Control.mixer.1 {
+					name 'Playback Volume 1'
+				}
+			}
+		}
+	}
+
+	Object.PCM {
+		pcm.$DEEP_BUFFER_PCM_ID {
+			name 'DeepBuffer'
```

Comment:
I think the name should be a parameters, e.g. 'Port0 DeepBuffer' or 'DeepBuffer for JackOut'

Reply:
thanks, changed